client/src/App.js

```diff
@@ -23,8 +23,8 @@ function App(props) {
         <Navbar />
         <main className="workspace">
           <Switch>
-            <Route exact path="/" component={UserProfile_LI} />
-            <Route exact path="/project" component={Project} />
+            <Route exact path="/:username" component={UserProfile_LI} />
```

Comment:
⚠️ using the params straight from / is gonna cause some issues with the other routes